src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
this is insufficient! think of how you can push more stuff to the parent class and use the make the child class cater to specific needs.